draw-canvas.js

```diff
@@ -1,37 +1,58 @@
-const {join} = require('path');
-const fs = require('fs').promises;
-const fetch = require('node-fetch');
+const { join } = require("path");
+const fs = require("fs").promises;
+const fetch = require("node-fetch");
 const InputParser = require("./input-parser.js");
 const Board = require("./board.js");
 const Options = require("./options.js");
 const Renderer = require("./renderer/index.js");
 const Grid = require("./grid.js");
 
+const base64Fetch = async (url) => {
+  const res = await fetch(url);
+  if (res.ok) {
+    const buffer = await res.buffer();
+    return `data:${res.headers.get("content-type")};base64,${buffer.toString(
+      "base64"
+    )}`;
+  }
+  const err = new Error(
+    `We couldn't seem to get our claws on the token image you asked for`
+  );
+  err.status = res.status;
+  throw err;
+};
+
 let fallbackTokenImage;
 const fetchTokenImageAsBase64 = async (code) => {
   if (!code) return null;
+
   try {
-    const url = `https://token.otfbm.io/img/${code}`;
-    const res = await fetch(url);
-    if (!res.ok) throw new Error('Non 200 status code');
-    const buffer = await res.buffer();
-    return `data:${res.headers.get("content-type")};base64,${buffer.toString("base64")}`
-  } catch(err) {
-    if (!fallbackTokenImage) {
-      const buff = await fs.readFile(join(__dirname, 'missing-token.jpg'));
-      fallbackTokenImage = buff.toString('base64');
+    return await base64Fetch(`https://token.otfbm.io/face/${code}`);
+  } catch (err) {
+    // swallow error and try img instead of face
+    if (err.status === 404) {
+      try {
+        return await base64Fetch(`https://token.otfbm.io/img/${code}`);
```

Comment:
I think `https://token.otfbm.io/zoom/${code}` will be better

Reply:
We decided against this in chat?